parent/pom.xml

```diff
@@ -337,6 +337,24 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
```

Comment:
I am now getting a ton of warning when building Forge or Openshift:

[WARNING] 'build.plugins.plugin.(groupId:artifactId)' must be unique but found duplicate declaration of plugin org.jboss.tools.tycho-plugins:repository-utils @ org.jboss.tools:parent:4.3.0.Alpha1-SNAPSHOT, /home/nboldt/.m2/repository/org/jboss/tools/parent/4.3.0.Alpha1-SNAPSHOT/parent-4.3.0.Alpha1-SNAPSHOT.pom, line 340, column 12